pkg/templates/scripts/guestfish/guestfish.sh.template

```diff
@@ -12,6 +12,15 @@ copy-device-to-device /dev/sdb /dev/sda
 # Move backup GPT data structures to the end of the disk
 part-expand-gpt /dev/sda
 
+# Remove root partition (content is redundant as we boot from recovery)
```

Comment:
just for future readers: please add that the root partition we delete here is the CoreOS root partition that the appliance does not use.

Reply:
done